README.md

```diff
@@ -62,28 +65,34 @@ Fully working examples of how to use these components are in `/example-crates`:
 [`electrum-client`]: https://docs.rs/electrum-client/
 [`bdk_chain`]: https://docs.rs/bdk-chain/
 
+## Dependencies
+
+- `bitcoind_rpc` depends on `bitcoind` being installed and available in `PATH` or in the `BITCOIND_EXEC` ENV variable.
+- `esplora` depends on [Blockstream's version of `electrs`](https://github.com/Blockstream/electrs)
+   being installed and available in `PATH` or in the `ELECTRS_EXEC` ENV variable.
+
```

Comment:
Worth putting a reference here to the Nix.md file and that using the Nix devshells will setup the bitcoind and electrs env.